### PR TITLE
Adding getting started file for EASE API

### PR DESCRIPTION
**Pull request by ksigler7**
- [x] Head data should have SEO title, keywords, and description
~- [ ] all images have an alt tag~
~- [ ] all images are named according to style guide~
- [x] Content is in Jobs-to-be-done format
- [x] Update all in-content links and links to deleted pages
- [ ] Added additional resources (aim for a link to the glossary, the blog and other jobs-to-be-done)
- [x] Added a TOC
- [ ] Added a talk to Dewey/Olivia section
- [x] title optimized for google search snippits
- [x] content optimized for google search snippits
- [x] keyword optimization for each category
- [x] pages are have the correct weight

### CODE REVIEW

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+---
+seo:
+  title: Getting Started with the Email Activity API (beta)
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: Getting Started with the Email Activity Feed API (beta)

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  description: Use the Email Activity query language to get started with the Email Activity API.
+  keywords: email activity, query language, email stats, email events
+title: Getting Started with the Email Activity API (beta)
+weight: 0
+layout: page
+navigation:
+  show: true
+---
+{% warning %}
+Email Activity is in beta and you may not have access - we are currently in the process of rolling out access. For more information, see the [Release Notes]({{root_url}}/release_notes.html#-Email-Activity-Feed-Beta). The feature may also change at any time without notice.
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: The Email Activity Feed is in beta...

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+- [Queries for common use cases](#-Queries-for-common-use-cases)
+- [Query reference](#-Query-reference)
+
+The API gives you access to query all of your stored messages, to query individual messages, and to download a CSV with data about the stored messages.
+
+{% anchor h2 %}
+Getting started
+{% endanchor %}
+
+Start with this basic query to the Email Activity API (replace `<<your API key>>` with an API key from your account):
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: +Start with this basic query to the Email Activity Feed API (replace `<<your API key>>` with an API key from your account):

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    <td><code>teammate</code> - teamates username</td>
+    <td><code>teammate="my_username"</code></td>
+  </tr>
+</table>
+
+{% anchor h2 %}
+Additional Resources
+{% endanchor h2 %}
+
+- [Email Activity API](https://sendgrid.api-docs.io/v3.0/email-activity/filter-all-messages)
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: Email Activity Feed API
Email Activity - Non Beta (no change)
Email Activity Feed - Beta

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 5]
```diff
@@ -16,6 +16,12 @@ Email Activity is in beta and you may not have access - we are currently in the
 - [Getting started](#-Getting-started)
     - [Encoding queries](#-Encoding-queries)
 - [Queries for common use cases](#-Queries-for-common-use-cases)
+  - [Filter by subject]()
+  - [Filter by reciepeint email]()
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: [filter by recipient email]

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 6]
```diff
@@ -16,6 +16,12 @@ Email Activity is in beta and you may not have access - we are currently in the
...
+  - [Filter by bounced emails]()
+- [Creating compound queries]()
+  - [Filter by a reciepeint email that was bounced]()
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: recipient

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 7]
```diff
@@ -76,15 +82,15 @@ Encoded, this query would look like this:
 Filter by reciepeint email
 {% endanchor %}
 
-Use this query to filter by a recipents email: (replace `<<your API key>>` with an API key from your account, and replace <<email>> with the recepients email):
+Use this query to filter by a recipents email: (replace `<<your API key>>` with an API key from your account, and replace <<email>> with the URL encoded recepients email):
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: URL encoded recipients email

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 8]
```diff
@@ -112,10 +118,72 @@ Encoded, this query would look like this:
...
 `status%3D%22bouced%22`
 
+{% anchor h2 %}
+Creating compound queries
+{% endanchor %}
+
+You may want to combine queries for a compound query. For example, you could filter for emails between a date range, or you could filter for when a specific recipients email is bounced. Here are some common use cases:
+
+{% anchor h3 %}
+Filter by a reciepeint email that was bounced
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: recipient

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 9]
```diff
@@ -112,10 +118,72 @@ Encoded, this query would look like this:
...
+{% endanchor %}
+
+Use this query to filter by a recipents email and by emails that are bounced: (replace `<<your API key>>` with an API key from your account, and replace <<email>> with the recepients email):
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: check both spellings of 'recipients' in this paragraph.

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 10]
```diff
@@ -112,10 +118,72 @@ Encoded, this query would look like this:
...
+Use this query to filter to emails between specific dates: (replace `<<your API key>>` with an API key from your account, and replace {start_date} and {end_date} with a UTC date string in this format: `YYYY-MM-DD HH:mm:SS`)
+
+{% codeblock %}
+curl --request GET \
+  --url 'http://api.sendgrid.com/v3/messages?limit=10&query>{start_date}<{end_date}' \
+  --header 'authorization: Bearer <<your API key>>'
+{% endcodeblock %}
+
+{% anchor h3 %}
+Filter by a reciepient and and a date range
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: Filter by a recipient and a date range

File: source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md [hunk 11]
```diff
@@ -112,10 +118,72 @@ Encoded, this query would look like this:
...
+{% endanchor %}
+
+Use this query to filter to emails by recepient and between specific dates: (replace `<<your API key>>` with an API key from your account, replace <<start_date>> and <<end_date>> with a UTC date string in this format: `YYYY-MM-DD HH:mm:SS`, and and replace <<email>> with the recepients email)
```
Comment (Whatthefoxsays) [inline source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md]: +Use this query to filter to emails by recipient and between specific dates: (replace `<<your API key>>` with an API key from your account, replace <<start_date>> and <<end_date>> with a UTC date string in this format: `YYYY-MM-DD HH:mm:SS`, and and replace <<email>> with the recipient's email)

**Comment by thinkingserious** [comment]
Hello @ksigler7,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**